terraform/modules/standards/outputs.tf

```diff
@@ -29,11 +23,17 @@ output "secondary_region" {
 }
 
 output "account_id" {
-  description = "The AWS account ID associated with the current caller identity"
+  description = "Deprecated. Use `aws_caller_identity.account_id`. The AWS account ID associated with the current caller identity"
   sensitive   = true
   value       = data.aws_caller_identity.this.account_id
 }
```

Comment:
Deprecating `account_id` to match how the platform module surfaces this. It might be interesting to start using the opentofu-specific `deprecation` fields once we're confident that all terraform is executed under opentofu....